src/Randomizer.SMZ3.Tracking/VoiceCommands/ChatIntegrationModule.cs

```diff
@@ -167,6 +178,75 @@ public async Task DeclareGanonsTowerGuessingGameWinner(int winningNumber)
             }
         }
 
+        /// <summary>
+        /// Asks the chat if tracker should increase content by one step
+        /// </summary>
+        /// <returns></returns>
+        public async Task AskChatAboutContent()
+        {
+            // Always ask the first time, otherwise it's a random change. Can probably change to always be random later.
+            var shouldAskChat = !HasAskedChatAboutContent || s_random.Next(0, 4) == 0;
+            if (!ShouldCreatePolls || !shouldAskChat)
+            {
+                Tracker.TrackItem(Tracker.Items.Where(x => x.Name.First() == "Content").First());
+                return;
+            }
+
+            AskChatAboutContentPollId = await ChatClient.CreatePollAsync("Do you think that was some high quality #content?", new List<string>() { "Yes", "No" }, AskChatAboutContentPollTime);
+
+            if (AskChatAboutContentPollId == null)
+            {
+                Tracker.TrackItem(Tracker.Items.Where(x => x.Name.First() == "Content").First());
+                return;
+            }
+
+            Tracker.Say(x => x.Chat.AskChatAboutContent);
+            Tracker.Say(x => x.Chat.PollOpened, AskChatAboutContentPollTime);
+            AskChatAboutContentCheckPollResults = true;
+            HasAskedChatAboutContent = true;
+            var task = Task.Run(async () =>
```

Comment:
If you purposefully want to "fire and forget" something, use an underscore `_ = Task.Run(…` to explicitly discard the results. Also, add a comment about this being on purpose.

Reply:
I've addressed this (and the others comments). Originally I was worried about locking things and preventing other things from running while waiting for the poll to complete, but never got around to confirming or denying that concern. Just verified that everything is good, so I updated everything to async/await instead of fire and forget.